pom.xml

```diff
@@ -252,6 +257,23 @@
                 <artifactId>jose4j</artifactId>
                 <version>${version.org.bitbucket.b_c.jose4j}</version>
             </dependency>
+            <dependency>
+                <groupId>io.undertow</groupId>
+                <artifactId>undertow-core</artifactId>
+                <version>2.3.0.Alpha2</version>
```

Comment:
Most of the artifacts have their versions defined in maven propertis. Can you do the same for these 3 new artifacts?

Reply:
done